src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/UpdateOrganisationTest.java

```diff
@@ -45,8 +45,10 @@ public void can_update_an_organisation_with_no_jurisdiction_should_return_400()
     }
 
 
-    // This test is for validating if old implementation i.e. Bearer token still works along with OPENID token as well since PRD needs to
-    // support both. Clients like EXUI is still using Bearer token and yet to migrate on OPENID. Currently all functional test cases are running
+    // This test is for validating if old implementation i.e. Bearer token still works along with OPENID token
+    // as well since PRD needs to
+    // support both. Clients like EXUI is still using Bearer token and yet to migrate on OPENID. Currently all
+    // functional test cases are running

```

Comment:
This comment would read better if it was split earlier, suggestion:

```
    // This test is for validating if old implementation i.e. 
    // Bearer token still works along with OPENID token as well 
    // since PRD needs to support both. 
    // Clients like EXUI is still using Bearer token and yet to migrate on OPENID. 
    // Currently all functional test cases are running via OPENID tokens.
```